app/models/language.rb

```diff
@@ -1,8 +1,7 @@
 class Language < ApplicationRecord
   include WorksOwner
-  validates_presence_of :short
-  validates :short, uniqueness: true
-  validates_presence_of :name
+  validates :short, presence: true, uniqueness: true, length: { maximum: 5 }
```

Comment:
Sorry I missed this last time, but per https://github.com/otwcode/otwarchive/blob/master/db/schema.rb#L638 the maximum should be 4 (the ticket has 5 or more characters as something that will cause an error)